internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -434,6 +434,21 @@ func createWindowsContainerDocument(ctx context.Context, coi *createOptionsInter
 		}
 	}
 
+	// Override WaitToKillServiceTimeout reg key to 30 minutes
```

Comment:
Would be nice to have a little more information here:
- Why is this needed?
- A reference to the comment in exec_hcs.go for further details on this behavior.
- An explanation of why we set it to 30 minutes (I think this was just arbitrarily chosen as "long enough that no one will want to wait longer", but that's good to document).